includes/class-newspack-listings-api.php

```diff
@@ -78,6 +79,71 @@ public static function register_routes() {
 				],
 			]
 		);
+
+		// GET listings taxonomy terms by name search term.
+		register_rest_route(
+			'newspack-listings/v1',
+			'children',
+			[
+				[
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => [ __CLASS__, 'get_children' ],
+					'permission_callback' => '__return_true',
+				],
+			]
+		);
+
+		// GET listings taxonomy terms by name search term.
```

Comment:
This should be updated - it's a POST request.

Reply:
Ah yes, updated in a3978bd